.redhat-ci.Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN dnf install -y \
         python-dbus \
         pylint \
         python-slip-dbus \
-        python-docker-py \
+        python2-docker \
```

Comment:
Hmm, if we go that way, I think I'd rather make it a new testsuite (they're delimited by `---` in the YAML). It's best if the F25/24 ones reflect as close as possible the actual state we release in.